react/components/EXPERIMENTAL_Charts/ScatterChart/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { FC } from 'react'
+import {
+  ScatterChart as ScatterChartBase, 
+  XAxis, 
+  YAxis, 
+  Tooltip,
+  Scatter,
+  CartesianGrid,
+  ResponsiveContainer,
+  ZAxis
+} from 'recharts'
+import PropTypes from 'prop-types'
+import uuid from 'uuid'
+import { commonDefaultProps } from './constants'
+import { getChartDefaultProps, getRangeOfZAxis } from '../helpers'
+import { colors } from '../commonProps'
+
+const CustomTooltip = (props) => {
+  return props.payload.map(item => 
+    <p key={uuid()}>{`${item.dataKey}: ${item.value}`}</p>
+  )
+}
+
+const ScatterChart:FC<BaseChartProps> = ({
+  data,
+  config,
+  xAxisKey,
+  yAxisKey,
+  zAxisKey
+}) => {
+  const { configs } = getChartDefaultProps(config, commonDefaultProps)
+  
+  return (
+    <ResponsiveContainer {...configs.container} >
+      <ScatterChartBase data={data}>
+        <CartesianGrid {...configs.grid}/>
+        <XAxis dataKey={xAxisKey} {...configs.xAxis} />
+        <YAxis dataKey={yAxisKey} {...configs.yAxis} />
+        {zAxisKey && 
+          <ZAxis dataKey={zAxisKey} range={getRangeOfZAxis(zAxisKey, data)}/>
+        }
+        <Tooltip cursor={false} content={<CustomTooltip />}/>
+        <Scatter fill={colors[0]}/>
+      </ScatterChartBase>
+    </ResponsiveContainer>
+  )
+}
+
+ScatterChart.propTypes = {
+  /** The source data, in which each element is an object. */
+  data: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
Don't you prefer to use `shape` instead of `object`?

Reply:
It's because this is the data, i can't suppose the shape of the objects, do you understand?